components/proxy/src/test/resources/logback.xml

```diff
@@ -7,7 +7,7 @@
     </encoder>
   </appender>
 
-  <root level="OFF">
+  <root level="DEBUG">
```

Comment:
An accidental change?
Do we leave it as DEBUG? Or if absolutely necessary can we keep it as INFO at least?

Reply:
Accidental.